evm/.gitignore

```diff
@@ -3,3 +3,4 @@ node_modules/
 build/contracts/*
 !contracts/vendor
 !build/contracts/Oracle.sol
+dist
```

Comment:
I commonly see this checked in. Is that an anti-pattern?

Reply:
yeah, any build artefacts should not be stored in version control. we should always have reproducible builds anyway